.gitignore

```diff
@@ -1,5 +1,5 @@
 __pycache__/
-
+config.*
```

Comment:
Heya. I know this is closed, but I just wanted to point out why you may have accidentally committed your twitch token. Because the directory and files are already committed, any changes to the files in the config directory will not be ignored unless you either remove the files or run:

 ```git update-index --assume-unchanged <file>```

Realistically I should have made some script of some sort to generate your config file locally instead of having it committed.

Reply:
thanks for the info, I've have already gotten my token changed so it being on here isn't an issue as its old.